src/common/models/Projection.ts

```diff
@@ -521,17 +517,16 @@ export class Projection {
       ] as MetricsTimeseriesRow;
       actualsTimeseries.push(actualsTimeseriesrowForDate || null);
       metricsTimeseries.push(metricsTimeseriesRowForDate || null);
-      dates.push(currDate.toDate());
+      // Clone the date since we're about to mutate it below.
+      dates.push(new Date(currDate.getTime()));
 
       // increment the date by one
-      currDate = currDate.clone().add(1, 'days');
+      currDate.setUTCDate(currDate.getUTCDate() + 1);
```

Comment:
Oh, do you mind adding a comment on this explaining that we're using native Date due to the issues you ran into with Luxon regarding daylight saving time?  That way somebody knows to be very careful if they rework this in the future.

Reply:
I'll add it at the top of the method.